homeassistant/components/nam/sensor.py

```diff
@@ -0,0 +1,94 @@
+"""Support for the Nettigo Air Monitor service."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import NAMDataUpdateCoordinator
+from .const import DOMAIN, SENSORS
+
+PARALLEL_UPDATES = 1
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Add a Nettigo Air Monitor entities from a config_entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+
+    sensors = []
+    for sensor in SENSORS:
+        if sensor in coordinator.data:
+            sensors.append(NAMSensor(coordinator, sensor))
+
+    async_add_entities(sensors, False)
+
+
+class NAMSensor(CoordinatorEntity, SensorEntity):
+    """Define an Nettigo Air Monitor sensor."""
+
+    coordinator: NAMDataUpdateCoordinator
+
```

Comment:
```suggestion
```

Reply:
Looks like mypy can't figure it out without it.

Reply:
Without this line `mypy` gives me those errors:
```
homeassistant/components/nam/sensor.py:75: error: "DataUpdateCoordinator[Any]" has no attribute "unique_id"  [attr-defined]
homeassistant/components/nam/sensor.py:80: error: Returning Any from function declared to return "DeviceInfo"  [no-any-return]
homeassistant/components/nam/sensor.py:80: error: "DataUpdateCoordinator[Any]" has no attribute "device_info"  [attr-defined]
```